src/components/ArtifactMilestonesPage/ArtifactMilestonesPage.tsx

```diff
@@ -0,0 +1,5 @@
+import { ContentBox } from 'components/ContentBox/ContentBox';
+
+export const ArtifactMilestonesPage = () => {
```

Comment:
This tab also contains Releases, shall we rename it to `ArtifactMilestonesReleasesPage`?

Reply:
According to the old UI there are only milestone based information with additional dates to be displayed.